spec/services/schedule_service_spec.rb

```diff
@@ -3,13 +3,17 @@
 RSpec.describe ScheduleService do
   describe "#schedule" do
     it "sets an edition's state to 'scheduled'" do
+      stub_default_publishing_api_put_intent
```

Comment:
Should we consider doing this in a before_each if we're writing it in most methods?

Reply:
Yep that makes sense! Originally I thought that because I was making a unique stub in one of the tests i'd get some sort of conflict, but as it turns out the unique stub overrides the before(:each)'s default stub so alls good :)